refinery/ui/source/js/user-file-browser/ctrls/user-file-browser-files-ctrl.js

```diff
@@ -95,13 +97,25 @@
     };
     vm.downloadCsvUrl = function () {
       return $location.protocol() + '://'
-          + $location.host() + ':' + $location.port()
-          + vm.downloadCsvPath();
+        + $location.host() + ':' + $location.port()
+        + vm.downloadCsvPath();
     };
     vm.gridOptions = gridOptionsService;
     vm.gridOptions.onRegisterApi = function (api) {
       api.core.on.sortChanged(null, vm.sortChanged);
     };
+
+    // helper method to create facet query for solr
+    function createFacetQuery () {
```

Comment:
shifted logic in service to use filter_attribute field which meant I had to separately make the fq field here